api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -484,10 +488,12 @@ message CertificateValidationContext {
   // certificate chain will be subject to validation by :ref:`CRL <envoy_v3_api_field_extensions.transport_sockets.tls.v3.CertificateValidationContext.crl>`.
   bool only_verify_leaf_cert_crl = 14;
 
-  // Config for the max number of intermediate certificates in chain that are parsed during verification.
-  // This does not include the leaf certificate. If configured, and the certificate chain is longer than allowed, the certificates
-  // above the limit are ignored, and certificate validation will fail. The default limit is 100,
-  // though this can be system-dependent.
-  // https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_verify_depth.html
+  // Defines maximum depth of a certificate chain accepted in verification. This number does not include the leaf, so a depth of 1 allows the leaf and one CA certificate.
+  // If a trusted issuer appears in the chain, but in a depth larger than configured, the certificate validation will fail.
+  // See `BoringSSL SSL_CTX_set_verify_depth <https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html#SSL_CTX_set_verify_depth>`
+  // If you use OpenSSL, its behavior is different from BoringSSL, this will define a limit on the number of certificates between the end-entity and trust-anchor certificates.
+  // Neither the end-entity nor the trust-anchor certificates count against depth. The default limit is 100, though this can be system-dependent.
```

Comment:
Is the default value is only for OpenSSL?
I'm guessing it's also for BoringSSL, so I suggest moving the default-value description before describing the differences between BoringSSL and OpenSSL in the comment.

Reply:
Yes, you're is right. Thanks.

FYI,
https://github.com/openssl/openssl/blob/7489ada9f3fd902c5bc3c58cc03a90de2800d0ab/crypto/x509/x509_vpm.c#L523
https://github.com/google/boringssl/blob/a1dffbfed9b78d77ab17c15fe2d189ca44704fd8/crypto/x509/x509_vpm.c#L519